packages/wrangler/src/__tests__/workflows.test.ts

```diff
@@ -480,6 +501,47 @@ describe("wrangler workflows", () => {
 		});
 	});
 
+	describe("instances send-event", () => {
+		const mockInstances: Instance[] = [
+			{
+				id: "foo",
+				created_on: mockCreateDate.toISOString(),
+				modified_on: mockModifiedDate.toISOString(),
+				workflow_id: "b",
+				version_id: "c",
+				status: "running",
+			},
+		];
+
+		const event = "my-event";
+
+		it("should send an event without payload to the bar instance given a name", async () => {
+			writeWranglerConfig();
+			await mockGetInstances(mockInstances);
+			await mockSendEventRequest("bar", event);
+
+			await runWrangler(
+				`workflows instances send-event some-workflow bar --type ${event}`
+			);
+			expect(std.info).toMatchInlineSnapshot(
+				`"📤 The event with type \\"my-event\\" and payload \\"{}\\" was sent to the instance \\"bar\\" from some-workflow"`
```

Comment:
Question/nit

> and payload \\"{}\\"

Should that part be removed altogether when there is "no" payload?
I guess you can better judge what's better for our users than I could